resources/zib/zib-Procedure-request.xml

```diff
@@ -109,20 +115,7 @@
         <rules value="open" />
       </slicing>
       <short value="ProcedureStartDate / ProcedureEndDate" />
-      <definition value="*   The (desired) start date (and if possible start time) of the procedure. A ‘vague’ date, such as only the year, is permitted. &#xD;&#xA;&#xD;&#xA;    The element offers the option to indicate the start of the period of a series of related procedures. &#xD;&#xA;&#xD;&#xA;*   The end date (and if possible end time) of the procedure. A ‘vague’ date, such as only the year, is permitted. &#xD;&#xA;&#xD;&#xA;    The element offers the option to indicate the end of the period of a series of related procedures. The end date element is only used for a procedures that takes some time and is then always applied. If the procedure still continues, the value is left empty. For instantaneous or very short lasting procedures the element is omitted." />
       <comment value="The `.occurrencePeriod.start` and `.occurrencePeriod.end` are used for ProcedureStartDate and ProcedureEndDate if the zib Procedure concerns a period. For instantaneous or very short lasting procedures the `.occurrenceDateTime` is used." />

```

Comment:
This `.comment` element should be removed (compare with `Procedure.performed[x]` in the zib-Procedure-event profile).

Reply:
Removed